lua/lsp/providers/yamlls.lua

```diff
@@ -0,0 +1,58 @@
+local opts = {
+  settings = {
+    yaml = {
+      hover = true,
+      completion = true,
+      validate = true,
+      schemaStore = {
+        enable = true,
+        url = "https://www.schemastore.org/api/json/catalog.json",
+      },
+      schemas = {
+        kubernetes = {
+          "daemon.yaml",
+          "manager.yaml",
+          "restapi.yaml",
+          "role.yaml",
+          "role_binding.yaml",
+          "*onfigma*.yml",
+          "*ngres*.yml",
+          "*ecre*.yml",
+          "*eployment*.yml",
+          "*ervic*.yml",
+          "*onfigma*.yaml",
+          "*ngres*.yaml",
+          "*ecre*.yaml",
+          "*eployment*.yaml",
+          "*ervic*.yaml",
+          "kubectl-edit*.yaml",
+        },
+        ["https://json.schemastore.org/ansible-role-2.9.json"] = {
+          "tasks/*.yml",
+          "tasks/*.yaml",
+        },
+        ["https://json.schemastore.org/ansible-inventory.json"] = {
+          "inventories/*.yml",
+          "*nventori*/*.yml",
+          "staging.yml",
+          "production.yml",
+        },
+        ["https://json.schemastore.org/ansible-playbook.json"] = {
+          "playbook.yml",
+          "playbook.yaml",
+          "main.yml",
+        },
```

Comment:
I'm not sure if it was available before, but there's now support for [ansiblels](https://github.com/williamboman/nvim-lsp-installer/blob/2f9a3800b976fb88f99a805dbb53f6ff6290a7ab/lua/nvim-lsp-installer/servers/ansiblels/init.lua). 

It would be good if you can give it a try if you already use ansible just to make sure there's conflict, since yamlls is set to validate the schemas.

Reply:
yeah it was not there beforehand, let me check, we need the kubernetes stuff though

Reply:
you are right, we don't need the ansible stuff anymore